doc/api/process.md

```diff
@@ -2222,11 +2222,27 @@ added: v0.9.12
 
 * {boolean}
 
-The `process.throwDeprecation` property indicates whether the
-`--throw-deprecation` flag is set on the current Node.js process. See the
-documentation for the [`'warning'` event][process_warning] and the
-[`emitWarning()` method][process_emit_warning] for more information about this
-flag's behavior.
+The `process.throwDeprecation` is a mutable property at runtime which
```

Comment:
This text is a little confusing.

* It's not clear what the difference between "mutable property at runtime" vs. "mutable property"  would be. I don't think there is any in this case.
* Since it is mutable, it doesn't necessarily indicate whether the command line flag is set.

Maybe something more like this?

> The initial value of `process.throwDeprecation` indicates whether the `--throw-deprecation` flag is set on the current Node.js process. See the documentation for the [`'warning'` event][process_warning] and
the [`emitWarning()` method][process_emit_warning] for more information. `process.throwDeprecation` is mutable, so whether or not deprecation warnings cause a process throw may be altered at runtime.

Reply:
Yeap, that's quite confusing.

Your proposal seems to be more clear than mine (on my opinion), but on my reading experience, I think is quite better switch the order of the sentences about `process.throwDeprecation` property and `emitWarning()` and `warning` event, to make that paragraph with a little bit more of cohesion.

Fixed on bf3bcf7.